File: App_Transportadoras-main/transportadores.dominio/entidades/Veiculo.cs


namespace transportadores.dominio.entidades;

public class Veiculo : EntidadeBase
{
    public string Placa { get; set; }
    public float Quilometragem { get; set; }
    public List<Peca> Pecas { get; set; }
    public string Chassi { get; set; }
    public Motorista? Motorista { get; set; }
    public int MotoristaId { get; set; }
    
}

File: App_Transportadoras-main/App/interfaces/veiculo.ts
import { EntidadeBase } from "./entidadeBase";
import { motorista } from "./motorista";

export interface Veiculo extends EntidadeBase {
    placa: string;
    quilometragem: number;
    chassi: string;
    motorista: motorista
    motoristaId: number;

}